src/app/store.js

```diff
@@ -1,10 +1,14 @@
-import { configureStore } from '@reduxjs/toolkit';
-import bookReducer from '../features/books/booksSlice';
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksState from '../features/books/booksReducer';
 
-const store = configureStore({
-  reducer: {
-    books: bookReducer,
-  },
+const reducers = combineReducers({
+  booksState,
 });
 
+const store = createStore(
```

Comment:
- [ ] This file should be in the redux folder.

![config](https://user-images.githubusercontent.com/61562626/149918584-cfb9d25c-734f-4272-b69a-c6257261a842.jpeg)